### ISSUE TITLE
Unable to empty the input

### ISSUE THREAD

**Issue by sumityadav** [kevinongko/vue-numeric#57]
I am using the min prop. 

On page load, the form is empty and the vue-numeric shows the input as empty (which is correct) but after the user key in any number, we do not have the option to empty the input again.

Here's the input:

```
<vue-numeric type="text" name="amount"
             placeholder="Amount"
             :min="5.00"
             currency="" 
             :precision=2 
             v-model="amount" 
             empty-value=""
></vue-numeric>
```

## Specifications

  - Plugin Version: 2.3.0
  - Vue.js Version: 2.5.13
  - Browser: Chrome Version 63.0.3239.132 (Official Build) (64-bit)
  - OS: MacOS 10.12.6

**Comment by mboboc** [comment]
I have the same problem and I observed that in the documentation it says `empty-string`'s type is `Number`. If `output-type` can be set to `String`, shouldn't `empty-string` also allow strings? At the moment it ignores `empty-string=""`. :confused: 

![Screenshot from 2022-03-17 14-54-25](https://user-images.githubusercontent.com/41815939/158813021-0b5755cc-56e9-4139-aed3-780e05dee75c.png)

**Comment by kevinongko** [comment]
Should be fixed in 2.5.0
please try again
